templates/solr.unit.j2

```diff
@@ -3,12 +3,12 @@ Description=Apache SOLR
 After=syslog.target network.target remote-fs.target nss-lookup.target
 
 [Service]
-PIDFile={{ solr_install_path }}/bin/solr-{{ solr_port }}.pid
-ExecStart={{ solr_install_path }}/bin/solr -e cloud -noprompt
+RemainAfterExit=yes
+Type=oneshot
 User=solr
-ExecReload=/bin/kill -s HUP $MAINPID
-ExecStop=/bin/kill -s QUIT $MAINPID
-PrivateTmp=true
+ExecStart=/etc/init.d/solr start
```

Comment:
One problem is I don't think I can guarantee the sysvinit files would always be available on systemd systems. For Solr 6 (and 5, I think, too), the `solr -e cloud -noprompt` is the correct way to start the service...